hack/configapi/cmd/delete.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/edgelesssys/constellation/v2/internal/api/attestationconfig/client"
+	"github.com/edgelesssys/constellation/v2/internal/staticupload"
+	"github.com/spf13/cobra"
+)
+
+// newDeleteCmd creates the delete command.
+func newDeleteCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "delete a specific version from the config api",
+		RunE:  runDelete,
+	}
+	cmd.PersistentFlags().StringP("version", "v", "", "Name of the version to delete (without .json suffix)")
+	must(enforceRequiredFlags(cmd, "version"))
+	return cmd
+}
+
+type deleteCmd struct {
+	attestationclient client.Client
+	closefn           client.CloseFunc
+}
+
+func (d deleteCmd) delete(ctx context.Context, cmd *cobra.Command) error {
+	defer func() {
+		if d.closefn != nil {
+			if err := d.closefn(ctx); err != nil {
+				fmt.Printf("close client: %s\n", err.Error())
+			}
+		}
+	}()
+	version := cmd.Flag("version").Value.String()
+	return d.attestationclient.DeleteAzureSEVSNVersion(ctx, version)
+}
```

Comment:
```suggestion
func (d deleteCmd) delete(cmd *cobra.Command) error {
	defer func() {
		if d.closefn != nil {
			if err := d.closefn(ctx); err != nil {
				cmd.Printf("close client: %s\n", err.Error())
			}
		}
	}()
	version := cmd.Flag("version").Value.String()
	return d.attestationclient.DeleteAzureSEVSNVersion(cmd.Context(), version)
}
```
* Don't need to pass an explicit context if the function already uses `*cobra.Command`, as the command already has a context embedded
* Use `cmd.PrintX` instead of `fmt.PrintX`